README.md

```diff
@@ -227,24 +232,24 @@ If the SDK has been provided with a valid secret and token the SSO user's channe
+The second is when the SDK is unable to verify the secret and/or token or encounters a network error. In either case, an _error view_ is shown. We recommend you handle all errors passed to the last argument of `CirrusMD.singleton.setToken`'s completion handler prior to showing the `CirrusMD.singleton.viewController` if possible. Doing so will provide a better experience for your user. Some errors may happen after the `CirrusMD.singleton.viewController` is already on screen. In that case, _error view_ is displayed.
 
-Two screens displayed by the SDK have default values that can be customized. The _logged out view_ and _error view_. We strongly recommend that you provide your own custom views for both cases. Because the CirrusMDSDK uses SSO to authenticate your patients, we are unable to provide logged out UI that helps the patient log back in. By providing your patients with a custom _logout out view_ you can, for example, provide relevant messaging and a button to log back in using the same SSO you implemented to log them in originally. Every time the _error view_ is shown the resolution is retrieving a new SSO token and setting it via `CirrusMDSDK.singleton.setToken(:)`. Providing a custom _error view_ gives you the ability to display relevant messaging and interactions the user can take, most likely a button to re-attempt SSO.
+Two screens displayed by the SDK have default values that can be customized. The _logged out view_ and _error view_. We strongly recommend that you provide your own custom views for both cases. Because the CirrusMDSDK uses SSO to authenticate your patients, we are unable to provide logged out UI that helps the patient log back in. By providing your patients with a custom _logout out view_ you can, for example, provide relevant messaging and a button to log back in using the same SSO you implemented to log them in originally. Every time the _error view_ is shown the resolution is retrieving a new SSO token and setting it via `CirrusMD.singleton.setToken(:)`. Providing a custom _error view_ gives you the ability to display relevant messaging and interactions the user can take, most likely a button to re-attempt SSO.
 
-Customization of both the _logged out view_ and _error view_ happens via the `CirrusMDSDKDelegate`.
+Customization of both the _logged out view_ and _error view_ happens via the `CirrusMDDelegate`.
 
-1. Implement `CirrusMDSDKDelegate`
+1. Implement `CirrusMDDelegate`
 2. Provide custom views for any or all of the optional delegates.
 3. Set a frame on the view you provide and it will be centered in the yellow area of the screen below.
 
 ##### Swift
 
 ```swift
-// CirrusMDSDKDelegate
+// CirrusMDDelegate
 
 func viewForError(code: CirrusMDSDKResult) -> UIView {
```

Comment:
I forget, did this class get renamed as well? (`CirrusMDSDKResult`)

Reply:
It did! Good catch.